share/revng/test/configuration/revng-qa/for-import-idb.yml

```diff
@@ -0,0 +1,31 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+tags:
+- name: ida
+  variables:
+    IFLAGS:
+      - -A
+      - -B
+    CFLAGS:
+      - -O0
+      - -g
+sources:
+- tags: [native, ida]
+  members:
+  - tests/model/import/idb/simple.c
+  - tests/model/import/idb/enum-type.c
+  - tests/model/import/idb/nested-struct.c
+  - tests/model/import/idb/qsort.c
+  - tests/model/import/idb/struct-as-field-ptr.c
+  - tests/model/import/idb/struct.c
+  - tests/model/import/idb/typedef-char-to-uint32.c
+commands:
+- type: revng-qa.idb
+  from:
+  - type: revng-qa.compiled
+    filter: ida
+  suffix: .i64
+  command: |-
+    unshare -rn ida64 -o$OUTPUT $IFLAGS $INPUT
```

Comment:
So this works assuming `ida64` is in `PATH`, correct?

Reply:
Yes... I am not sure how are we going to ensure that.

Reply:
I'll take care of this.